src/store/selectors/index.js

```diff
@@ -0,0 +1,44 @@
+import { useSelector } from 'react-redux';
+import bookPlaceholderSvg from 'assets/book-placeholder.svg'
+
+const loadingBook = {
+  title: 'Loading...',
+  author: 'loading...',
+  coverImageUrl: bookPlaceholderSvg,
+  publisher: 'Loading Publishing',
+  synopsis: 'Loading...',
+  loadingBook: true,
+};
+
+const loadingBooks = Array.from({length: 10}, (v, index) => ({
+  id: `loading-book-${index}`,
+  ...loadingBook,
+}));
```

Comment:
Should use object creator to generate those initial states -> we should not use the same object for this.

Reply:
Thank you, refactored.

Reply:
Thank you, refactored.